src/constants/water-indicators.js

```diff
@@ -1,58 +1,175 @@
+export const IDS = {
+  bws: {
+    baseline: '8074ac9b-9cca-4aaf-a112-26166a8e9c7d',
+    projectedAbsolute: '6f339b41-ea2f-4502-a454-d03bb22b540b',
+    projectedChange: 'd48cf5e5-8692-4096-a711-4604f9be163c',
+    legacy: {
+      baseline: '1a1d4f61-f1b3-4c1a-bfb5-9d0444ecdd56',
+      projectedAbsolute: '935f9a49-a45a-4362-b9f7-7a8e22df5146',
+      projectedChange: 'd8439b5e-c7f0-4021-9347-f1e68ef8122e',
+    },
```

Comment:
Do we need to keep around the legacy config? It's in the git history, and we _probably_ won't need it, right?